internal/jujuapi/applicationoffers.go

```diff
@@ -178,11 +178,20 @@ func (r *controllerRoot) modifyOfferAccess(ctx context.Context, change jujuparam
 		if err := r.jimm.PermissionManager().GrantOfferAccess(ctx, r.user, change.OfferURL, ut, change.Access); err != nil {
 			return errors.E(op, err)
 		}
+		// We grant access on the controller because pre 3.6.5 controllers are consulting their database to check if
```

Comment:
It should be pre 3.6.6 I think because 3.6.5 is already out and has the same logic of checking the DB.

Reply:
done